package.json

```diff
@@ -27,13 +27,13 @@
   "license": "Apache-2.0",
   "dependencies": {
     "bpk-tokens": "^36.0.0",
-    "lodash": "^4.17.20",
+    "lodash": "^4.17.21",
```

Comment:
Unrelated, but simple to pull the patch change in here and save an extra release.